config/set/celebrity/celebrity.yaml

```diff
@@ -32,8 +32,7 @@ services:
     Rector\Celebrity\Rector\FuncCall\SetTypeToCastRector: ~
 
     # class { var $value; } → class { public $value; }
-    Rector\Php52\Rector\Property\VarToPublicPropertyRector: ~pu
-
+    Rector\Php52\Rector\Property\VarToPublicPropertyRector: ~
```

Comment:
@gnutix This was the issue :D slow PHPStorm + my hasty hands

Reply:
Ahahah that's a nice one xD Too bad it's still "valid YAML"... can't catch this with a linter.

Reply:
Indeed :D 

But now it's not valid for the configuration - it will throw exception here now: https://github.com/rectorphp/rector/pull/2043/files#diff-4b356c6550ab7f23c4b4d17595032977R69